packages/@react-aria/combobox/src/useComboBox.ts

```diff
@@ -199,19 +200,24 @@ export function useComboBox<T>(props: AriaComboBoxProps<T>, state: ComboBoxState
     }
 
     lastValue.current = state.inputValue;
-  }, [state.selectionManager, state.inputValue]);
+  }, [state.selectionManager, state.inputValue, allowsCustomValue, state.focusStrategy, state.selectedKey]);
+
+  // TODO: ask what else needs to go into the aria-labelledby for the button (difference between current spec and aria 1.2)
+  let triggerLabel = props['aria-labelledby'] || labelProps.id;
```

Comment:
@majornista what is this the correct way to label the combobox trigger button? Should I follow the v2 convention of concatting the textfield's id and the trigger's id and having an `aria-label` that mentions how many suggestions are available?